### PR TITLE
fix(unit-tests): fixed unit tests

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: test/services/helper/helper.spec.js [hunk 1]
```diff
@@ -17,14 +17,16 @@ describe("helper functionality", function () {
       const mockWorkflowObject = require("../../dataset/create.json");
       const mockFields = ["sky"];
       const result = helper.objectContainsFields(mockWorkflowObject, mockFields);
-      expect(result).to.be.false;
+      expect(result).to.be.a("string");
+      expect(result).to.not.be.undefined;
```
Comment (asafMasa) [inline test/services/helper/helper.spec.js:21]: check if one of the conditions is disposable

File: test/services/helper/helper.spec.js [hunk 2]
```diff
@@ -17,14 +17,16 @@ describe("helper functionality", function () {
...
+
     });
     it("Should return false if object is not contains the correct fields", function () {
```
Comment (asafMasa) [inline test/services/helper/helper.spec.js:24]: false or undefined?

File: test/services/helper/helper.spec.js [hunk 3]
```diff
@@ -17,14 +17,16 @@ describe("helper functionality", function () {
       const mockWorkflowObject = require("../../dataset/create.json");
       const mockFields = ["sky"];
       const result = helper.objectContainsFields(mockWorkflowObject, mockFields);
-      expect(result).to.be.false;
+      expect(result).to.be.a("string");
```
Comment (asafMasa) [inline test/services/helper/helper.spec.js:20]: check that unit tests it("message") defines the expected results properly

File: test/services/workflows/ingestWorkflow.spec.js [hunk 4]
```diff
@@ -7,6 +7,7 @@ const Helper = require("../../../services/helper/helper");
 const IngestWorkflow = require("../../../services/workflows/ingestWorkflow");
 const logger = require("../../../logger/logger");
 const jWorkflow = require("jWorkflow");
+const { MCLogger } = require("@map-colonies/mc-logger");
```
Comment (asafMasa) [inline test/services/workflows/ingestWorkflow.spec.js:10]: it's not in use

File: services/workflows/baseWorkflow.js [hunk 5]
```diff
@@ -25,27 +25,21 @@ module.exports = class BaseWorkflow {
...
-            callback: (result) => {
-              if (result instanceof Error) {
-                return reject(result);
-              }
-              return resolve(this._returnValue);
-            },
-            initialValue: "",
-          });
+      return await workflowOrder.start({
+        callback: (result) => {
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply services/workflows/baseWorkflow.js]: fixed in fdb2851

File: services/workflows/baseWorkflow.js [hunk 6]
```diff
@@ -25,27 +25,21 @@ module.exports = class BaseWorkflow {
...
-          workflowOrder.start({
-            callback: (result) => {
-              if (result instanceof Error) {
-                return reject(result);
-              }
-              return resolve(this._returnValue);
-            },
-            initialValue: "",
-          });
+      return await workflowOrder.start({
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply services/workflows/baseWorkflow.js]: fixed in fdb2851

File: test/services/helper/helper.spec.js (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply test/services/helper/helper.spec.js:22]: fixed in 7ad3409

File: test/services/helper/helper.spec.js (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply test/services/helper/helper.spec.js]: fixed in 7ad3409

File: test/services/workflows/ingestWorkflow.spec.js (hunk 4 above)
Comment (CL-SHLOMIKONCHA) [reply test/services/workflows/ingestWorkflow.spec.js]: fixed in 7ad3409

File: test/services/helper/helper.spec.js (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply test/services/helper/helper.spec.js]: fixed in 7ad3409

**Pull request merged**